### PR TITLE
wip: inline staking

### PR DESCRIPTION
**Pull request by 0xKheops**

### CODE REVIEW

File: apps/extension/src/ui/domains/Staking/useStakingBanner.tsx [hunk 1]
```diff
@@ -137,24 +83,22 @@ export const useStakingBanner = () => {
 
       if (lsdTokenBases.includes(token.id))
         result = evmLsdStakingTokenEligible({ token, addresses })
-      else result = nomPoolStakingTokenEligible({ token, addresses })
+      else result = false
```
Comment (chidg) [inline apps/extension/src/ui/domains/Staking/useStakingBanner.tsx]: `result` is already false

File: apps/extension/src/ui/domains/Staking/NomPoolBond/useNomPoolBondWizard.ts [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+  const maxPlancks = useMemo(() => {
+    if (!balance || !existentialDeposit || !fakeFeeEstimate) return null
+    // use 11x fake fee estimate as we block form based on 10x the real fee estimate
+    if (existentialDeposit.planck + fakeFeeEstimate * 11n > balance.transferable.planck) return null
+    return balance.transferable.planck - existentialDeposit.planck - fakeFeeEstimate * 11n
+  }, [balance, existentialDeposit, fakeFeeEstimate])
+
+  const inputErrorMessage = useMemo(() => {
+    if (isSoloStaking)
+      return t("An account cannot do both regular staking and nomination pool staking")
```
Comment (chidg) [inline apps/extension/src/ui/domains/Staking/NomPoolBond/useNomPoolBondWizard.ts]: Requested by Toby: can we make the element containing this message a bit smaller to make it break more evenly? Or add a line break half way through the message?

**Review by chidg — CHANGES_REQUESTED** [review]
It's very beautiful!
Couple of small changes/comments.

File: apps/extension/src/ui/domains/Staking/useStakingBanner.tsx (hunk 1 above)
Comment (0xKheops) [reply apps/extension/src/ui/domains/Staking/useStakingBanner.tsx]: true!

**Review by chidg — APPROVED** [review]
LGTM!

**Pull request merged**